src/transactions/types.ts

```diff
@@ -131,8 +131,8 @@ export interface NftTransfer {
   transactionHash: string
   timestamp: number
   block: string
-  fees: Fee[]
-  nfts: Nft[]
+  fees?: Fee[]
+  nfts?: Nft[]
```

Comment:
While we're at it, it might be good to add an optional `metadata?` field to NFT transfers to keep it in-sync with blockchain-api. Not needed though.